bundles/pubsub/pubsub_admin_tcp/src/pubsub_tcp_topic_sender.c

```diff
@@ -198,7 +195,7 @@ pubsub_tcp_topic_sender_t *pubsub_tcpTopicSender_create(
                 char *url = NULL;
                 asprintf(&url, "tcp://%s:%u", bindIP, port);
                 char *bindUrl = NULL;
-                asprintf(&bindUrl, "tcp://0.0.0.0:%u", port);
+                asprintf(&bindUrl, "tcp://127.0.0.1:%u", port);
```

Comment:
Why change this to localhost instead of 0.0.0.0?

Reply:
this is an error, i have fixed this.